### PR TITLE
Update zh-cn translation

### PR DESCRIPTION
**Pull request by Rad586**

### CODE REVIEW

File: docs/zh-cn/blocks.md [hunk 1]
```diff
@@ -28,11 +29,11 @@ description: 方块相关调整
 
 这里是一些额外的设置项：
 
-* 随机化孵化时间，大幅随机化孵蛋器的孵化时间
+* 随机化孵化时间，大幅随机化孵蛋器的孵化时间。
 
 ::: details 自定义行为
 
-从 1.9.0 起，孵蛋器支持生成随机实体，设置 NBT，以及在孵化完成时执行命令。
+从 1.9.0 起，孵蛋器支持生成随机实体，设置 NBT，以及在孵化完成时执行命令。1.10.0 后，`/` 样式的命令被[命令官模组的命令](https://constellation-mc.github.io/commander/Commands)取代了，并且 `time` 支持使用[表达式](https://constellation-mc.github.io/commander/Expressions)。
```
Comment (melontini) [inline docs/zh-cn/blocks.md:36]: The link should point to zh-cn wiki.

File: docs/zh-cn/mechanics.md [hunk 2]
```diff
@@ -56,104 +57,64 @@ description: 一些机制，还有一些数不上名的小玩意
...
+现在有一个新的选项：僵尸能否投掷物品？
 
 若启用，僵尸将能够投掷它们所拾起的可投掷物品。
 
 ::: details 添加自定义投掷行为
 数据包能做的很有限，只能执行命令和生成带颜色的粒子（就像墨囊和染料那样）。
 
-所有自定义行为需要和 `recipes`, `tags`, `loot_tables` 一块，放置在数据包的 `andromeda/item_throw_behavior` 文件夹下。可以任取自定义行为的具体文件名。
+所有自定义行为需要和 `recipes`, `tags`, `loot_tables` 一块，放置在数据包的 `andromeda/item_throw_behavior` 文件夹下，文件名任取。
+1.10.0 版本开始，这个特性通过[命令官](https://modrinth.com/mod/cmd)模组来实现其功能。你可以在它的维基上了解到命令、表达式等的用法。https://constellation-mc.github.io/commander/
```
Comment (melontini) [inline docs/zh-cn/mechanics.md:68]: Same here.

File: docs/zh-cn/mechanics.md [hunk 3]
```diff
@@ -56,104 +57,64 @@ description: 一些机制，还有一些数不上名的小玩意
...
-`disabled`：禁用该物品的所有自定义投掷行为。
+| 事件  |   |
+|---|---|
+| `block`  | 击中方块时执行  |
+| `entity`  | 击中实体时执行  |
+| `miss`  | 错过目标时执行  |
+| `any`  | 在以上所有事件之后执行  |
 
-`complement`：若为 false，自定义的行为将覆盖它原本的行为，若为 true，则将在它执行完原本的行为前执行自定义行为。
+`commands` 是一系列[命令官模组中的命令](https://constellation-mc.github.io/commander/Commands)。
```
Comment (melontini) [inline docs/zh-cn/mechanics.md:108]: And here.

**Review by melontini — CHANGES_REQUESTED** [review]

**Comment by Rad586** [comment]
Done! Thanks for the heads-up!

**Review by melontini — APPROVED** [review]

**Pull request merged**